jina/peapods/zmq/__init__.py

```diff
@@ -215,6 +218,7 @@ def close(self):
             This method is idempotent.
 
         """
+        self.is_closing = True
```

Comment:
Are we sure this is necessary? When u close, there won't be any more data being received. Another thing is if u need to consume the latest messages in a buffer ...

Reply:
If it is closing, it should not send an idle request anymore. That is the only thing I wanted to adress.

Reply:
But since this is blocking, this could never happen right? I am not sure about it. Have u seen the problem?

Reply:
yes if it is blocking then we don't need this. I removed it

Reply:
Well, I am not sure, is the zmq closed when the Runtime receives terminate signal or when the Pea closes the process?